runtime/src/bank.rs

```diff
@@ -3579,6 +3594,30 @@ impl Bank {
         self.log_epoch_rewards_sysvar("update");
     }
 
+    /// Create the persistent PDA containing the epoch-rewards data
+    fn create_epoch_rewards_partition_list_account(
```

Comment:
```suggestion
    fn create_epoch_rewards_partition_metadata_account(
```
or
```suggestion
    fn create_epoch_rewards_partition_receipt_account(
```

we kinda rugged the name dropping the commitment list